### DISCUSSION TITLE
proper way to override CSS for a component

### DISCUSSION THREAD

**Discussion by nickcom** [rsuite/rsuite#1906] [Q&A]
Sorry for the noob question here, but what is the preferred way to override the Rsuite styling. For example if I want to make the background-color of my <Navbar> yellow? Or to make it's height 75px?

I see you can customise a theme. However I simply want to edit a component individually.

**Comment by SevenOutman** [comment]
You may wrap your custom styles in a CSS class and apply it to the component. Most of RSuite components accept a `className` prop for adding additional classes to their root element.
